app.js

```diff
@@ -0,0 +1,36 @@
+let arrayOfTodos = []
+
+const fetchTodos = () => {
+  fetch('https://jsonplaceholder.typicode.com/todos')
+    .then((response) => response.json())
+    .then((json) => arrayOfTodos = json)
+}
+
+const logTodos = () => {
+  console.log(arrayOfTodos)
+}
+
+const populateTodos = () => {
+  for (let i = 0; i < arrayOfTodos.length; i++) {
+    let e = arrayOfTodos[i];
+    let li = document.createElement('li');
+
+    let userId = "UserID = " + e.userId + ', ';
+    let id = "ID = " + e.id + ', ';
+    let title = 'Title = ' + e.title + ', ';
+    let completed = "Completed = " + e.id;
+
+    li.append(userId, id, title, completed)
+
+    // if (e.completed == true) { li.className = "green" }
+    // else (li.className = "red")
+
+    li.className = e.completed === true ? 'green' : 'red';
```

Comment:
nice! you can even remove the `=== true` bc `e.completed' value is either **True** or **False**.  so it would translate to `li.className = true ? 'green' : 'red'`; or `li.className = false ? 'green' : 'red'`; so you can shorten this to `li.className = e.completed ? 'green' : 'red'`